testem.js

```diff
@@ -1,8 +1,11 @@
-/*jshint node:true*/
-module.exports = {
+/* eslint-env node */
+
+let config = {
   "framework": "mocha",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
+  "ignore_missing_launchers": true,
+  "parallel": 3,
```

Comment:
On second thought, it could be a good idea to implement something like a `testem.local.js` file where you can override values of the `testem.js` file, so we can ditch the env variables. Shouldn't be too hard.